rewrite-java-tck/src/main/java/org/openrewrite/java/tree/ConstructorTest.java

```diff
@@ -173,4 +174,127 @@ public A(String a) {
           )
         );
     }
+
+    @Issue("https://openjdk.org/jeps/513")
+    @MinimumJava25
```

Comment:
Could it make sense to add these four new tests to a nested test class that has these two annotations? That way we're not repeating ourselves and can run those selectively.